cdisc_rules_engine/operations/library_model_column_order.py

```diff
@@ -79,15 +79,15 @@ def _get_variables_metadata_from_standard_model(
 
         if domain_details:
             # Domain found in the model
-            class_name = domain_details["_links"]["parentClass"]["title"]
+            class_name = domain_details["_links"]["parentClass"]["title"].upper()
```

Comment:
`Special-Purpose` should map to `SPECIAL PURPOSE`. I _think_ this is the only special case.

I would also externalize this to a utility method like `convert_library_to_ct_class` because we will likely need this elsewhere.